services/movie_session.py

```diff
@@ -0,0 +1,80 @@
+from datetime import datetime
+from typing import List
+
+from django.utils.dateparse import parse_date
+
+from db.models import CinemaHall, Movie, MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id,
+    )
+
+
+def get_movies_sessions(session_date: datetime = None) -> List[MovieSession]:
+    """
+    Retrieve movie sessions, optionally filtered by a specific date.
+    """
+    # If a session date is provided, parse it and filter sessions
+    if session_date:
+        date = parse_date(session_date)
```

Comment:
The `parse_date` function expects a string, but `session_date` is typed as `datetime`. Consider changing the type of `session_date` to `str` if you intend to parse it from a string, or adjust the logic if you are working with `datetime` objects.